dmk2cw.c

```diff
@@ -813,13 +830,16 @@ main(int argc, char** argv)
 	    rx02_data = 2 + (256 << dmk_track[idamp+4]);
 	    /* Follow CRC with one RX02-MFM FF */
 	    dmk_track[fmtimes + datap + rx02_data++] = 0xff;
-	  }
+	  } else {
+            /* Compute expected sector size */
+            sector_data = 2 + (128 << dmk_track[idamp+4]) + extra;
```

Comment:
Why doesn't this line's computation match up with the implementation of `secsize()` in `cw2dmk.c`?

Is this the new sector end detection not going to work for non-IBM  FM formatted disks?  What about high order bits with MFM?

[Not related to this PR's changes] Does current secsize() implementation undercompute size when QUIRK_EXTRA_CRC and/or QUIRK_EXTRA?